### ISSUE TITLE
focused panel display is broken in multi-window mode

### ISSUE THREAD

**Issue by mieldepoche** [godotengine/godot#81158]
### Godot version

4.1.1

### System information

linux

### Issue description

[simplescreenrecorder-2023-08-30_14.27.36.webm](https://github.com/godotengine/godot/assets/71937306/f2151dea-b6d4-41e4-b3ff-9e5b9110ccfa)

[simplescreenrecorder-2023-08-30_14.28.40.webm](https://github.com/godotengine/godot/assets/71937306/1894167d-391a-4716-857e-80e7fc40b4ca)

as you can see in multi window mode the unfocused docks still have their focus border visible

### Steps to reproduce

1. make docks floating
2. click on them
3. notice the focus border doesn't disapear when a dock is not focused anymore

### Minimal reproduction project

any

**Comment by Sauermann** [comment]
I'm wondering, if this should be considered a bug or a feature.

Current behavior is, that each separate native window remembers the dock that has focus.
So with a Alt-Tab I can switch to the other native window and the previously focused dock still has focus.

To me this looks like a reasonable behavior, which should diverge from single-window-mode, where all docks are in a single native Window.

**Comment by mieldepoche** [comment]
>  So with a Alt-Tab I can switch to the other native window and the previously focused dock still has focus.

I agree this should still happen, but the actual focus indicator should not be visible in unfocused windows imo. There only ever 1 focused dock at any time, so multiple docks should not be displayed as focused. So alt-tabbing would work as it currently work while updating the focus indicator. (as a side note, I don't really see in which context you'd alt-tab between floating docks seeing how few things you can do with the keyboard alone in them :thinking: )
